scss-test.scss

```diff
@@ -0,0 +1,19 @@
+.unsorted {
+  color: #FFFFFF;
+  background: red;
+  .inception-1 {
+    color: #DDD;
+    .inception-2 {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3